### ISSUE TITLE
German eszett () Wght 900 | opsz 14

### ISSUE THREAD

**Issue by EbenSorkin** [TypeNetwork/Roboto-Flex#259]
I was thinking about how the 'ß' begins to heavily distort its proportions:

<img width="69" alt="image" src="https://user-images.githubusercontent.com/2181213/156049078-14b8ea17-f993-42da-9c42-34948a871e87.png">
<img width="92" alt="image" src="https://user-images.githubusercontent.com/2181213/156049100-22824cd5-96dc-463b-9247-b1f3640f93e7.png">

And I thought why not pull the stroke back? Three wide strokes at the base HAVE to be that wide. But if it is more like two and bit of a flip like at the base of the letter would not have to be so wide.

I also went to see "how atypical this solution I am objecting to?". In other words - maybe it is more common than I think. But I am not finding that's not true. 

<img width="141" alt="image" src="https://user-images.githubusercontent.com/2181213/156051604-79337e20-f9ec-4dc3-96c8-6baa9a6780d0.png">
<img width="132" alt="image" src="https://user-images.githubusercontent.com/2181213/156051676-2c460bbe-a349-483c-8b68-bcee0641301f.png">
<img width="128" alt="image" src="https://user-images.githubusercontent.com/2181213/156051734-0adf9574-9ece-43c4-9b93-e07eee077ca6.png">
<img width="122" alt="image" src="https://user-images.githubusercontent.com/2181213/156051777-49e2c24f-a619-4b3e-9b3b-a4a284e3222c.png">

It exists, but not as extreme as this. And where the feature of the exists the stroke or serif is made smaller. Which brings up another point which is that when you do see something like this it seems to be more connected to a serif design.
<img width="127" alt="image" src="https://user-images.githubusercontent.com/2181213/156051852-9e6f6b30-4e62-4c09-943a-f396c6bf0f22.png">
<img width="134" alt="image" src="https://user-images.githubusercontent.com/2181213/156051945-636550e8-eb01-4ca5-9683-79dffce7aeba.png">

So in summary think it makes better sense in order to preserve the intent of the source Roboto to change either just the black or maybe all versions by cutting the inner stroke short so that it is possible to keep the general proportions rather than the stroke extent.
